### PR TITLE
Fectch a poster and tests are complete

### PR DESCRIPTION
**Pull request by jdavalos98**
Fetching a sing poster is functional and all tests are passing. Fixed error where it was finding 8 posters instead of 4. I also added the coverage file to our git ignore file so it doesn't get pushed with each added feature.

### CODE REVIEW

**Comment by jdavalos98** [comment]
I believe that after this PR the git ignore will kick in but did just look and those files were pushed up.

File: config/routes.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
   get "/api/v1/posters", to: "api/v1/posters#index"
 
+  get "/api/v1/posters/:id", to: "api/v1/posters#show"
```
Comment (jchirch) [inline config/routes.rb:10]: Looks straightforward and functional, I'll use this in my testing file for the next route I'm working on.

File: app/controllers/api/v1/posters_controller.rb [hunk 2]
```diff
@@ -5,6 +5,11 @@ def index
         render json: PosterSerializer.render_all_posters(Poster.all)
     end
 
+    def show
```
Comment (jchirch) [inline app/controllers/api/v1/posters_controller.rb:8]: Great use of a serializer, reflects our recent class lesson.

File: .gitignore [hunk 3]
```diff
@@ -31,3 +31,4 @@
```
Comment (jchirch) [inline .gitignore:31]: Thank you for updating the git/ignore file, cleans up the app and would save processing time if this were a more robust application.

**Review by jchirch — APPROVED** [review]
Looks great, as a side note thank you for figuring out that double database error that was impacting our count.

**Pull request merged**